packages/circuits/src/PreventDoubleActionProof.ts

```diff
@@ -0,0 +1,99 @@
+import { Circuit, Prover } from './circuits'
+import { SnarkProof } from '@unirep/utils'
+import { BigNumberish } from '@ethersproject/bignumber'
+import { BaseProof } from './BaseProof'
+
+/**
+ * The reputation proof structure that helps to query the public signals
+ */
+export class PreventDoubleActionProof extends BaseProof {
+    readonly idx = {
+        epochKey: 0,
```

Comment:
This mapping is not correct, it includes outputs for a different circuit and doesn't include outputs for this circuit.